src/libstore/download.cc

```diff
@@ -369,11 +369,13 @@ struct CurlDownloader : public Downloader
 
         curlm = curl_multi_init();
 
-        #if LIBCURL_VERSION_NUM >= 0x072b00 // correct?
```

Comment:
The answer to your question is that it was indeed correct, @edolstra :)
https://curl.haxx.se/libcurl/c/CURLMOPT_PIPELINING.html

I can remove/reformat/change the comments if you prefer.